units/commit_queue.go

```diff
@@ -114,13 +138,17 @@ func (j *commitQueueJob) Run(ctx context.Context) {
 		return
 	}
 	if cq.Processing {
+		processingSeconds := time.Since(cq.ProcessingUpdatedTime).Seconds()
 		grip.Info(message.Fields{
 			"source":             "commit queue",
 			"job_id":             j.ID(),
 			"item_id":            nextItem.Issue,
 			"project_id":         cq.ProjectID,
-			"processing_seconds": time.Since(cq.ProcessingUpdatedTime).Seconds(),
+			"processing_seconds": processingSeconds,
 		})
+		if processingSeconds > 3600 {
```

Comment:
I know we originally said doing this every time the job ran would be too much, but one hour might actually be too long: perhaps every 10 minutes? 🤔 Or at least 30 minutes. I'm just making these numbers up do whatever makes sense to you, we can change it later :)

Reply:
removed based on in person conversation